test/models/thinker_test.rb

```diff
@@ -1,7 +1,49 @@
 require 'test_helper'
 
 class ThinkerTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+ 	def setup
+    @thinker = Thinker.new(name: "Example Thinker", email: "thinker@example.com")
+  end
+
+  test "should be valid" do
+    assert @thinker.valid?
+  end
+
+  test "name should be present" do
+    @thinker.name = "     "
+    assert_not @thinker.valid?
+  end
+
+  test "email should be present" do
+  	@thinker.email = "     "
```

Comment:
A small detail, but I've noticed that there's a mix of tabs and spaces which is leading to slightly confusing indents in Github (which appears to use 4 spaces for a tab). You might like to look at how your IDE is configured. Typically it's going to be easier to ensure consistency by using 'soft tabs' (which expand to a couple of spaces).